APCAppCore/APCAppCore/UI/Onboarding/APCIntroVideoViewController.m

```diff
@@ -102,6 +104,9 @@ - (void) playbackDidFinish:(NSNotification*)notification {
 #pragma mark - Public Methods
 
 - (void) dismiss {
+    // reset the preferred orientation mask before dismissing
+    [[APCAppDelegate sharedAppDelegate] setPreferredOrientationMask:0];
```

Comment:
So... I realize that I wrote this but now I'm thinking it would maybe be better to define a reset method?

````
- (void)resetPreferredOrientationMask
````